lib/importproject.cpp

```diff
@@ -1497,3 +1505,20 @@ bool ImportProject::sourceFileExists(const std::string &file)
 {
     return Path::isFile(file);
 }
+
+void ImportProject::setFileSizes()
+{
+    for (auto &fs : fileSettings) {
+        int statResult;
+#ifdef _WIN32
+        struct _stat statBuf;
+        statResult = _stat(fs.filename().c_str(), &statBuf);
```

Comment:
This is not very elegant. I have the feeling we should have some kind of getFileSize utility functionality instead. Somewhere we get file sizes for the filenames listed in mFiles and I rather reuse the code that figures out the filesize.

Reply:
That logic is internal to `FileLister`. But maybe it could be moved to `FileWithDetails`?